### PR TITLE
Improving logs

### PR DESCRIPTION
**Pull request by macartur**
Set logging configuration using  etc/kytos/logging.ini
Fixed syslog

### CODE REVIEW

File: etc/kytos/logging.ini [hunk 1]
```diff
@@ -1,38 +1,46 @@
...
-args: []
-formatter: brief
- 
+args:[sys.stdout]
+formatter: thread
+
 [handler_syslog]
 class: handlers.SysLogHandler
-args: [('localhost', handlers.SYSLOG_UDP_PORT), handlers.SysLogHandler.LOG_USER]
+args: ['/dev/log']
```
Comment (diraol) [inline etc/kytos/logging.ini]: @macartur this should be `/var/log` doesn't it?

File: kytos/core/logs.py [hunk 2]
```diff
@@ -27,14 +27,14 @@ def add_stream_handler(cls, stream):
...
-        handler = logging.StreamHandler(stream)
+        handler = StreamHandler(stream)
         cls._add_handler(handler)
         return handler
 
     @staticmethod
     def _add_handler(handler):
-        handler.setFormatter(logging.Formatter(FORMAT))
-        logging.getLogger().addHandler(handler)
+        handler.setFormatter(Formatter(FORMAT))
```
Comment (diraol) [inline kytos/core/logs.py]: Can't you retrieve the format from one of those defined on the `logging.ini` file?

**Review by diraol — CHANGES_REQUESTED** [review]

File: etc/kytos/logging.ini (hunk 1 above)
Comment (macartur) [reply etc/kytos/logging.ini]: This is the device used to write in local syslog. I will try to change this.

Comment (macartur) [reply etc/kytos/logging.ini]: I am using that http://scottbarnham.com/blog/2008/01/01/sysloghandler-not-writing-to-syslog-with-python-logging/, I can use syslog using logging.ini yet.

Comment (cemsbr) [reply etc/kytos/logging.ini]: I just noticed that I have /dev/log :). I did a quick read in the link above and also in Python's docs and I thinks it's ok.

Comment (diraol) [reply etc/kytos/logging.ini]: Agreed!

**Comment by diraol** [comment]
@macartur also do not forget to mention which issue you are dealing with on you commit(s) message(s) =)

**Comment by beraldoleal** [comment]
@macartur waiting for reply on @diraol comment.

**Review by diraol — APPROVED** [review]

**Pull request merged**